README.md

```diff
@@ -20,33 +23,35 @@ These instructions can be used for AWS:
     docker run --rm -v $PWD:$PWD:Z -w $PWD quay.io/coreos/tectonic-builder:bazel-v0.3 bazel --output_base=.cache build tarball
     ```
 
-2. Extract the tarball
+1. Extract the tarball
```

Comment:
I prefer keeping this numbered list accurate. Since I tend to read the source a lot more often than I read the rendered result.

Reply:
> I prefer keeping this numbered list accurate.

Rerolled (on top of #56, which I've also updated based on this comment) with dff2718 -> bc625f4.